frontend/src/pages/UpdateUser.tsx

```diff
@@ -1,5 +1,139 @@
-import { Box } from '@mui/material';
+import {
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+  Button,
+  FormControl,
+} from '@mui/material';
+import Background from '../components/Background';
+import { useState } from 'react';
+import { Faculty } from '../types';
 
 export const UpdateUser = () => {
-  return <Box>This is the UpdateUser Page!</Box>;
+  const [userDetails, setUserDetails] = useState<{
+    emojis: string;
+    faculty: string;
+    faveEat: string;
+    interests: string;
+  }>({ emojis: '', faculty: '', faveEat: '', interests: '' });
+
+  // TODO:
+  // Add appropriate type to event
+  const handleChange = (e: any) => {
+    const { name, value } = e.target;
+    setUserDetails({
+      ...userDetails,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async () => {
+    console.log(userDetails);
+    // TODO:
+    // Hit the correct API to successfully store info to the db
+    const res = await fetch('http://localhost:3001/updateUser', {
+      method: 'POST',
+    });
+    await res.json();
+  };
+
+  return (
+    <Background bgcolor={'#fff'}>
```

Comment:
Rather than a background, I think this will look better with a TopNavBar. There's the added benefit that the user can also logout without having to complete the update user page